src/SectionsNavigation.Abstractions/SectionsNavigation.md

```diff
@@ -82,20 +82,39 @@ You can observe the state with the `ISectionsNavigator.StateChanged` event.
 If you invoke 2 operations simultaneously (double tap, press 2 buttons with 2 fingers, etc.), only the first will actually run.
 This is because the request state (Processing, Processed or FailedToProcess) is part of the `ISectionsNavigator.State`.
 If a request is made while another is processing, the second request is cancelled.
+This is logged as a warning.
 
 ### Background navigation
-You can navigate in sections that are not active. This is useful if you want to _prepare_ a section before entering it.
+You can navigate in sections that are not active.
+This is useful if you want to _prepare_ a section before entering it.
 
 ### Stack navigator adapter
-You can use `SectionsNavigatorToStackNavigatorAdapter` to adapt a `ISectionsNavigator` into a `IStackNavigator`. This is useful if your app mostly use the single stack pattern.
-
-### Customize animations
-When using `FrameSectionsNavigator`, you can customize or remove the animations of the MultiFrame using `MultiFrame.Animations`.
+You can use `SectionsNavigatorToStackNavigatorAdapter` to adapt a `ISectionsNavigator` into a `IStackNavigator`.
+This is useful if your app mostly use the single stack pattern.
+
+### Transitions and Animations
+When using `FrameSectionsNavigator`, you can customize or remove the animations using `SectionsTransitionInfo`.
+You can specify transition info per request (using `SectionsNavigatorRequest.TransitionInfo`) and globally using the following:
```

Comment:
Can we specify what the default transition do for each platform if we don't precise them?
like IOS: (slide from left => right?
like Android: (slide from left => right?
Maybe in the future add to gif animation?